src/pages/api/sanity/lessons/create.ts

```diff
@@ -0,0 +1,101 @@
+import {NextApiRequest, NextApiResponse} from 'next'
+import {sanityClient} from 'utils/sanity-client'
+import {getTokenFromCookieHeaders} from 'utils/auth'
+import fetchEggheadUser from 'api/egghead/users/from-token'
+import {nanoid} from 'nanoid'
+
+type LessonData = {
+  title: string
+  fileMetadata: {
+    fileName: string
+    signedUrl: string
+  }
+}
+
+type SanityVideoResource = {
+  _type: string
+  _id: string
+  filename: string
+  originalUrl: string
```

Comment:
I'm not stoked on this and think it might be `sourceVideoUrl` or something a little more descriptive

`mezzanine files` is a term of art referring to these files, but that's probably too esoteric for general use.

`originalVideoUrl` might be a happy middle ground?

Reply:
this isn't a blocker, just thinking about names and what we will live with over time.